xmodule/modulestore/draft_and_published.py

```diff
@@ -136,8 +141,11 @@ def _flag_publish_event(self, course_key):
             if bulk_record and bulk_record.active:
                 bulk_record.has_publish_item = True
             else:
+                print(f"reload damn you")
                 # We remove the branch, because publishing always means copying from draft to published
                 self.signal_handler.send("course_published", course_key=course_key.for_branch(None))
```

Comment:
Add a filter here to make sure course_id is not an instance of LibraryLocator before sending to the event bus

Reply:
I used `course_key.is_course` based on https://github.com/openedx/opaque-keys/blob/03254a909eb428b86eda0b897e03ed6ea87aa600/opaque_keys/edx/keys.py#L25 -- seem right?